lib/segment/src/index/field_index/geo_index/mod.rs

```diff
@@ -544,7 +544,11 @@ impl PayloadFieldIndex for GeoMapIndex {
         None
     }
 
-    fn estimate_cardinality(&self, condition: &FieldCondition) -> Option<CardinalityEstimation> {
+    fn estimate_cardinality(
+        &self,
+        condition: &FieldCondition,
+        _hw_counter: &HardwareCounterCell, // Nothing to measure here.
```

Comment:
Are you sure about this? I see that at least we read `points_of_hash` from the index, and we have mutable, immutable, and mmap variants